sdk/core/azure_core_test/src/proxy/client.rs

```diff
@@ -77,9 +74,9 @@ impl Client {
         let mut url = self.endpoint.clone();
         url = url.join("/Record/Stop")?;
         let mut request = Request::new(url, Method::Post);
-        request.insert_header("accept", "application/json");
-        request.insert_header("content-type", "application/json");
-        request.insert_header(X_RECORDING_ID, recording_id.to_string());
+        request.insert_header(ACCEPT, "application/json");
+        request.insert_header(CONTENT_TYPE, "application/json");
+        request.insert_header(RECORDING_ID, recording_id.to_string());
```

Comment:
Nit: For me its clearer if the Keys were named HEADER_*

Reply:
That would be redundant given they are all defined in the `header` module. If you like being explicit, import `typespec_client_core::http::header` and you can use `header::ACCEPT`. That's the idiomatic way. We don't like unnecessary repetition.